docs/tools/pd-control.md

```diff
@@ -406,42 +424,66 @@ Usage:
 ```bash
 >> region                               //　Display the information of all regions
 {
-  "count": 1,
-  "regions": [...]
+  "count": 10,
+  "regions": [
+    {
+      "id": 4,
+      "start_key": "",
+      "end_key": "t\\x80\\x00\\x00\\x00\\x00\\x00\\x00\\xff\\x05\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\xf8",
+      "epoch": {...},
+      "peers": [...],
+      "leader": {...},
+      "written_bytes": 251302,
+      "read_bytes": 60472,
+      "approximate_size": 1,
+      "approximate_keys": 367
+    },
+    ...
+  ]
 }
 
 >> region 2                             // Display the information of the region with the id of 2
 {
-  "region": {
-      "id": 2,
-      ...
-  }
-  "leader": {
-      ...
-  }
+  "id": 2,
+  "start_key": "t\\x80\\x00\\x00\\x00\\x00\\x00\\x00\\xff\\x1d\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\xf8",
+  "end_key": "",
+  "epoch": {...},
+  "peers": [...],
+  "leader": {...},
+  "written_bytes": 251302,
+  "read_bytes": 60472,
+  "approximate_size": 96,
+  "approximate_keys": 524442
 }
 ```
 
-### `region key [--format=raw|pb|proto|protobuf] <key>`
+### `region key [--format=raw|encode] <key>`
```

Comment:
We may need to add `hex` here?

Reply:
@disksing What's your opinion about `hex`? @rleungx Would you please tell me exactly what the heading should look like if we do need it? Thanks!

Reply:
em, I think we need to update pingcap/docs first to include features about hex.